CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# Wie du mitmachen kannst
+
+## Code Formatierung
+Wir nutzen [Black](https://black.readthedocs.io/en/stable/#) zum automatisierten Formatieren von Python Code.
+Black kannst du auch [für deine Entwicklungsumgebung einrichten](https://black.readthedocs.io/en/stable/editor_integration.html).
```

Comment:
Missing usage documentation here
Please add: `How to install requirements txt?` and `how to run black for all files, maybe a git hook if possible?`

Reply:
Maybe also a short note on `venv` ?

Reply:
I think we should switch to [pipenv](https://pipenv.readthedocs.io) to make things easier.

Reply:
A git hook sounds great! 😍